src/core/server/stacks/approveComment.ts

```diff
@@ -69,4 +71,33 @@ const approveComment = async (
   return result.after;
 };
 
+const enableRepliesToChildren = async (
+  commentID: string,
+  mongo: MongoContext
+) => {
+  const children = await mongo
+    .comments()
+    .find({
+      parentID: commentID,
+    })
+    .toArray();
+
+  const nonRejectedChildIDs = children
```

Comment:
Should we update to only do the rest of this if there are `children`?

Reply:
I am seeing this as resolved, but I don't see a guard clause (or similar check) to block if `!children || children.length === 0`. Am I missing something?

Reply:
I've pushed a change for this, but this conversation is not being marked as "Outdated", and the change isn't showing up here, but if you just look at the code on this branch on Github you should now see that check

Reply:
https://github.com/coralproject/talk/blob/fix/CORL-2261-children-of-rejected-comments/src/core/server/stacks/approveComment.ts#L85